tools/skyline2isocor/skyline2isocor.xml

```diff
@@ -0,0 +1,39 @@
+<tool id='skyline2isocor' name='Skyline2Isocor: converting skyling outputs into IsoCor inputs' version='@TOOL_VERSION@+galaxy0' profile='23.0'>
```

Comment:
The profile is maybe a little high. It can be an issue if a Galaxy not-really-up-to-date plan to install your tool.

I can see that on IUC, they seem to stick to `20.01` or `20.05`. I guess that they aren't any feature on your tool that require something special.

Reply:
Yes true, I will stick with the IUC recommendations then. Thanks